src/txn/txn_rollback_nondurable.c

```diff
@@ -145,15 +145,14 @@ __txn_abort_newer_updates(
 		 * changes - and thus won't need to roll back structure
 		 * changes on internal pages.
 		 */
+		/* That is a lie: I think we need to check for fast deleted pages */
 		break;
 	case WT_PAGE_ROW_LEAF:
 		__txn_abort_newer_row_leaf(session, page, rollback_timestamp);
 		break;
 	default:
-		/* TODO: Fix this */
-		WT_RET_MSG(session, EINVAL,
-		    "rollback_nondurable_commits only supports row store");
-
+		WT_RET_MSG(session, EINVAL, "rollback_nondurable_commits "
+		    "is only supported for row store btrees");
```

Comment:
Why limit this to row-store? Off-hand I can't think of any real difference between row- and column-store for this functionality. (I'd also drop the word "btree", LSM, column-store, they're all Btrees.)